opentelekomcloud/resource_opentelekomcloud_rds_instance_v3.go

```diff
@@ -0,0 +1,1004 @@
+				Required: true,
+				ForceNew: true,
+			},
+
+			"backup_strategy": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Optional: true,
+				ForceNew: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"start_time": {
+							Type:     schema.TypeString,
+							Required: true,
+							ForceNew: true,
+						},
+						"keep_days": {
+							Type:     schema.TypeInt,
+							Computed: true,
+							Optional: true,
+							ForceNew: true,
+						},
+					},
+				},
+			},
+
+			"ha_replication_mode": {
+				Type:     schema.TypeString,
+				Computed: true,
+				Optional: true,
+				ForceNew: true,
+			},
+
+			"param_group_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
+
+			"created": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"nodes": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"availability_zone": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"name": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"role": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"status": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+					},
+				},
+			},
+
+			"private_ips": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+			},
+
+			"public_ips": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+			},
+		},
+	}
+}
+
+func resourceRdsInstanceV3UserInputParams(d *schema.ResourceData) map[string]interface{} {
+	return map[string]interface{}{
+		"terraform_resource_data": d,
+		"availability_zone":       d.Get("availability_zone"),
+		"backup_strategy":         d.Get("backup_strategy"),
+		"db":                      d.Get("db"),
+		"flavor":                  d.Get("flavor"),
+		"ha_replication_mode":     d.Get("ha_replication_mode"),
+		"name":                    d.Get("name"),
+		"network_id":              d.Get("network_id"),
+		"param_group_id":          d.Get("param_group_id"),
+		"security_group_id":       d.Get("security_group_id"),
+		"volume":                  d.Get("volume"),
+		"vpc_id":                  d.Get("vpc_id"),
+	}
+}
+
+func resourceRdsInstanceV3Create(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+	client, err := config.sdkClient(GetRegion(d, config), "rdsv1", serviceProjectLevel)
+	if err != nil {
+		return fmt.Errorf("Error creating sdk client, err=%s", err)
+	}
+	client.Endpoint = strings.Replace(client.Endpoint, "/rds/v1/", "/v3/", 1)
+
+	opts := resourceRdsInstanceV3UserInputParams(d)
+	opts["region"] = GetRegion(d, config)
```

Comment:
Can we still specify the region on the resource?

Reply:
no need to specify it.